css/style.css

```
@@ -0,0 +1,41 @@
+body {
+}
+
+.head {
+  display: flex;
+  justify-content: center;
+}
+
+h1 {
+  border: 2px solid black;
+  border-radius: 10px;
+  padding: 5px 100px;
+}
+
+.card-body {
+  border: 2px solid red;
+  height: 700px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 2px solid blue;
+  margin: 15px;
+  margin-top: 25px;
```

You should better combine this (should work like this, maybe you need to add a fourth `15px` value for the last side):
```suggestion
  margin: 25px 15px 15px;
```

You could also use `margin-inline` or `margin-block`, [have a look here on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/margin#see_also).